docs/usage/shoot_scheduling_profiles.md

```diff
@@ -0,0 +1,69 @@
+# Shoot Scheduling Profiles
+
+This guide describes the available scheduling profiles and how they can be configured in the Shoot cluster. It also clarifies how a custom scheduling profile can be configured.
+
+## Scheduling profiles 
+
+The scheduling process in the [kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) happens in series of stages. A [scheduling profile](https://kubernetes.io/docs/reference/scheduling/config/#profiles) allows configuring the different stages of the scheduling.
+
+As of today, Gardener supports two predefined scheduling profiles:
+
+- `balanced` (default)
+
+   **Overview** 
+
+   The `balanced` profile attempts to spread Pods evenly across Nodes to obtain a more balanced resource usage. This profile provides the default kube-scheduler behavior.
+   
+   **How it works?**
+   
+   The kube-scheduler is started without any profiles. In such case, by default, one profile with the scheduler name `default-scheduler` is created. This profile includes the default plugins. If a Pod doesn't specify a scheduler name, kube-apiserver sets it to `default-scheduler`. Then, the Pod gets scheduled by the `default-scheduler` accordingly.
+  
+- `bin-packing` (alpha)
+
+   **Overview**
+
+   The `bin-packing` profile scores Nodes with a high allocation of resources higher than the others, resulting in a greater overall resource utilizaton. By favoring the Nodes with higher allocation some of the other Nodes become underutilized over time (because new Pods keep being scheduled to the most allocated Nodes). Then, the cluster autoscaler identifies such underutilized Nodes and removes them from the cluster. In this way this profile provides a greater overall resource utilization (compared to the `balanced` profile).
```

Comment:
```suggestion
   The `bin-packing` profile scores Nodes with a high allocation of resources higher than the others, resulting in a greater overall resource utilizaton. By favoring the Nodes with higher allocation, some of the other Nodes become under-utilized over time (because new Pods keep being scheduled to the most allocated Nodes). Then, the cluster-autoscaler identifies such under-utilized Nodes and removes them from the cluster. In this way, this profile provides a greater overall resource utilization (compared to the `balanced` profile).
```